templates/login/login.html

```diff
@@ -0,0 +1,24 @@
+{% extends "base.html" %}{% load static %}{%block content%}
+<div id="login">
+  <h3 class="text-center text-white pt-5">Login form</h3>
+  <div class="container">
+    <div id="login-row" class="row justify-content-center align-items-center">
+      <div id="login-column" class="col-md-6">
+        <div id="login-box" class="col-md-12">
+          <form id="login-form" class="form" action="" method="post">
+              <div class="container">
+                <h3 class="text-center text-info">Login</h3>
+              </div>
+              {% csrf_token %}
+              {{ form.as_p }}
+              <div class="form-group">
+                <input type="submit" name="submit" class="btn btn-info btn-md" value="submit">
```

Comment:
maybe it will be nice to replace the value to something else - maybe "login"?
What do you think?

Reply:
> maybe it will be nice to replace the value to something else - maybe "login"? What do you think?

I'll change it tnx